docroot/modules/custom/nvli_custom_search/src/Plugin/Block/ResourceTypeTitleBlock.php

```diff
@@ -47,6 +47,9 @@ public function build($resource_type = NULL) {
       '#title' => $title,
       '#resource_id' => $filterID,
       '#resource_link' => NULL,
+      '#cache' => array(
```

Comment:
@dishabhadra I think this can be cached? Based on url context? The only thing that changes i believe is the url when the keyword is available in the url.